packages/lib-classifier/src/hooks/useTranscriptionReductions.js

```diff
@@ -26,8 +46,21 @@ export default function useTranscriptionReductions() {
 
   const { loaded, caesarReductions } = useCaesarReductions(workflow.caesarReducer)
 
+  useEffect(function checkTranscriptionUsers() {
+    async function checkSubject() {
+      const authorization = await getBearerToken(authClient)
+      const alreadySeen = await checkUser(caesarReductions?.userIDs, authorization)
+      if (alreadySeen) {
+        subject.markAsSeen()
```

Comment:
Maybe run this after logging the error, so that we can inspect the original value of `already_seen` in Sentry.